### ISSUE TITLE
training loss = 0 auc_test = None

### ISSUE THREAD

**Issue by Chailchai** [FunctionLab/sei-framework#25]
Hello, I want to apply your sei-model to my own data set. According to the help file you posted, I installed the corresponding version of the software. First, I conducted a test run based on the data you provided. Due to the large amount of data , I only filtered the sequence information of chr1 for training, and modified epoch=1000 to see the training results as soon as possible, but the program returned the following information. Why is this? Please answer the following, thank you.

![image](https://github.com/FunctionLab/sei-framework/assets/108219248/e03919a9-a09f-4b4d-bb16-b7550c51decd)
